extensions/amp-story-subscriptions/0.1/test/test-amp-story-subscriptions.js

```diff
@@ -1,37 +1,75 @@
 import '../amp-story-subscriptions';
-import {htmlFor} from '#core/dom/static-template';
+import * as Preact from '#core/dom/jsx';
+
+import {afterRenderPromise} from '#testing/helpers';
+
+import {registerServiceBuilder} from '../../../../src/service-helpers';
+import {
+  Action,
+  AmpStoryStoreService,
+} from '../../../amp-story/1.0/amp-story-store-service';
+import {AmpStorySubscriptions} from '../amp-story-subscriptions';
 
 describes.realWin(
   'amp-story-subscriptions-v0.1',
   {
     amp: {
       runtimeOn: true,
-      extensions: ['amp-story-subscriptions:0.1'],
+      extensions: ['amp-story:1.0', 'amp-story-subscriptions:0.1'],
     },
   },
   (env) => {
     let win;
     let doc;
-    let html;
+    let subscriptionsEl;
+    let storeService;
+    let storySubscriptions;
+
+    const nextTick = () => new Promise((resolve) => win.setTimeout(resolve, 0));
 
     beforeEach(() => {
       win = env.win;
       doc = win.document;
-      html = htmlFor(doc);
+
+      storeService = new AmpStoryStoreService(win);
+      registerServiceBuilder(win, 'story-store', function () {
+        return storeService;
+      });
+
+      subscriptionsEl = (
+        <amp-story-subscriptions layout="container"> </amp-story-subscriptions>
+      );
+      const storyEl = doc.createElement('amp-story');
+      storyEl.appendChild(subscriptionsEl);
+      doc.body.appendChild(storyEl);
+
+      storySubscriptions = new AmpStorySubscriptions(subscriptionsEl);
     });
 
     it('should contain amp-subscriptions attributes', async () => {
-      const element = html`
-        <amp-story-subscriptions layout="container"> </amp-story-subscriptions>
-      `;
-      doc.body.appendChild(element);
-      await element.whenBuilt();
+      await subscriptionsEl.whenBuilt();
       expect(
-        element.querySelector('div').hasAttribute('subscriptions-dialog')
+        subscriptionsEl
+          .querySelector('div')
+          .hasAttribute('subscriptions-dialog')
       ).to.equal(true);
       expect(
-        element.querySelector('div').getAttribute('subscriptions-display')
+        subscriptionsEl
+          .querySelector('div')
+          .getAttribute('subscriptions-display')
```

Comment:
Is this attribute coming from `amp-subscriptions`?

Reply:
Yes. You can see more details in their API doc here: https://amp.dev/documentation/components/amp-subscriptions